mdoc-sbt/src/main/scala/mdoc/MdocPlugin.scala

```diff
@@ -167,7 +173,8 @@ object MdocPlugin extends AutoPlugin {
           MdocJSConfiguration(
             scalacOptions = options.options,
             compileClasspath = options.classpath,
-            linkerClassPath = getJars(linkerDependency) ++ workerClasspath,
+            linkerClassPath =
+              getJars(linkerDependency) ++ workerClasspath ++ getJars(importMapDependency),
```

Comment:
Should we always add importMapDependency? Or can we detect if we need it from config?

Reply:
I think this would desirable, but I believe to be a large amount of work.

If I've understood correctly, the mdoc plugin writes out the classpath it's going to feed mdocJS at build time. 

Evidently, the tests don't know at build time, whether or not they will include the CLI flag, so i think it has to stay included here. I _believe_ that it should be possible to still run the CLI without this dependency. 

It is however, a very good point that the current documentation would not work with the import map functionality. I will create another issue (and pinky promise to follow up with a docs PR) that updates the classpath generation here;
<https://scalameta.org/mdoc/docs/js.html#command-line>